scripts/data_files/driver_jsons/driver_opaque_schema.json

```diff
@@ -2,25 +2,31 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
   "properties": {
-    "prefix": {
+    "_comment": {
       "type": "string"
     },
+    "prefix": {
+      "type": "string",
+      "pattern": "^[A-Z_a-z][0-9A-Z_a-z]*$"
+    },
     "type": {
-      "type": "string"
+      "type": "string",
+      "const": ["opaque"]
     },
     "location": {
-      "type": "string"
+      "type": ["integer","string"],
+      "pattern": "^(0x|0X)?[a-fA-F0-9]+$"
     },
-    "dependency": {
+    "mbedtls/h_depend_on": {
       "type": "string"
     },
     "headers": {
       "type": "array",
-      "items": [
-        {
+      "items": {
           "type": "string"
-        }
-      ]
+        },
+        "minItems": 1,
+        "uniqueItems": true
```

Comment:
Why should header names be unique? Not that it wouldn't be surprising to have duplicates, but I don't understand why we'd enforce this constraint.

Goes for both types of drivers.